src/test/java/seedu/eduke8/topic/TopicTest.java

```diff
@@ -0,0 +1,67 @@
+package seedu.eduke8.topic;
+
+import org.junit.jupiter.api.Test;
+import seedu.eduke8.common.Displayable;
+import seedu.eduke8.exception.Eduke8Exception;
+import seedu.eduke8.hint.Hint;
+import seedu.eduke8.option.Option;
+import seedu.eduke8.option.OptionList;
+import seedu.eduke8.question.Question;
+import seedu.eduke8.question.QuestionList;
+import java.util.ArrayList;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+class TopicTest {
+
+    private static final String QUESTION_ONE_DESCRIPTION = "First Question";
+    private static final String QUESTION_TWO_DESCRIPTION = "Second Question";
+    private static final String QUESTION_THREE_DESCRIPTION = "Third Question";
+    private static final String HINT = "Hint";
+    private static final String INPUT = "JUnit: Basic";
+
+    @Test
+    void getsTopicDescription_topicDescription_returnsTopicDescription() throws Eduke8Exception {
```

Comment:
Just some minor details, would the first part of the naming be better to follow the naming in Topic? getDescription_... Similar to getsQuestionList_questionList_returnsQuestionList( )